README.md

```diff
@@ -66,6 +66,18 @@ In order to translate enum values for editor, define following in language XML f
 </languages>
 ```
 
+#### UseUnderlyingTypeAttribute
+
+This attribute can be used on enum properties which are using `EnumEditorDescriptor` or `EnumSelectionFactory` in order to change how `SelectItem` is sent to Episerver UI.
+
+By default, enum values are serialized to json using a configured converter. In some cases, this might be undesirable and by adding `UseUnderlyingType` one can override this behavior.
+
+```cs
+    [EditorDescriptor(EditorDescriptorType = typeof(EnumEditorDescriptor<EnumType>))]
```

Comment:
Just a comment but I don't have  good solution:
Now  [UseUnderlyingType] can be used standalone. It cana only be used with [EditorDescriptor(EditorDescriptorType = typeof(EnumEditorDescriptor<EnumType>))] which is kind-of tricky.

Ideally we whould have a different version of EnumEditorDescriptor (E.g. UnderlyingTypeEnumEditorDescriptor) but it leads to other problems (e.g. how to implemente NullableEnumEditorDescriptor and it's second version).

Since I don;t have a good solution markiing this as OK solution